wrapper/src/vmaf.h

```diff
@@ -63,6 +63,10 @@ class StatVector
 public:
     StatVector() {}
     StatVector(std::vector<double> l): l(l) {}
+    StatVector clear()
```

Comment:
What does this mean?

Reply:
We define a public method to the StatVector class, which allows us to erase all the contents of a created StatVector object. This is used when we re-use the StatVector perModelScore, which fills up with predictions from each bootstrap model and then is re-used for the next model.